packages/access-client/src/agent.js

```diff
@@ -43,43 +42,49 @@ function emailToSessionPrincipal(email) {
 }
 
 /**
- * @param {Ucanto.Signer<Ucanto.DID<'key'>>} space
+ * @param {Ucanto.Signer<Ucanto.DID<'key'>>} issuer
+ * @param {Ucanto.DID} space
  * @param {Ucanto.Principal<Ucanto.DID<'mailto'>>} account
  * @returns
  */
-async function createSpaceSaysAccountCanAdminSpace(space, account) {
+async function createSpaceSaysAccountCanAdminSpace(issuer, space, account) {
   return ucanto.delegate({
-    issuer: space,
+    issuer,
     audience: account,
     capabilities: [
       {
         can: 'space/*',
-        with: space.did(),
+        with: space,
       },
       {
         can: 'store/*',
-        with: space.did(),
+        with: space,
       },
       {
         can: 'upload/*',
-        with: space.did(),
+        with: space,
       },
     ],
   })
 }
 
 /**
- * @param {Ucanto.Signer<Ucanto.DID<'key'>>} space
+ * @param {Ucanto.Signer<Ucanto.DID<'key'>>} issuer
+ * @param {Ucanto.DID} space
  * @param {Ucanto.Principal<Ucanto.DID<'key'>>} device
  */
-async function createSpaceSaysDeviceCanAccessDelegateWithSpace(space, device) {
+async function createSpaceSaysDeviceCanAccessDelegateWithSpace(
```

Comment:
may want to call this function now/soon `createIssuerSays...` since the authorization will now no longer always be from Space.